pallets/messages/src/tests.rs

```diff
@@ -363,6 +425,61 @@ fn invalid_payload_location_onchain() {
 	});
 }
 
+#[test]
+fn add_ipfs_message_with_large_payload_errors() {
+	new_test_ext().execute_with(|| {
+		let caller_1 = 5u64;
+
+		let old_max_size = MaxMessagePayloadSizeBytes::get();
+		MaxMessagePayloadSizeBytes::set(10u32);
```

Comment:
Tests run in parallel. So this might fail some tests intermittently

https://stackoverflow.com/questions/51694017/how-can-i-avoid-running-some-tests-in-parallel

Reply:
Hmm.... that would mean we'd have to mark pretty much every test in that particular suite to run serially (most of them, anyway). Unless you know of a way to override the mocked value of that pallet config value for just a single test?

Reply:
Reworked--found a middle-ground value for the constant that satisfies all conditions under test.